x-pack/plugins/monitoring/public/views/elasticsearch/ccr/index.js

```diff
@@ -30,6 +30,9 @@ uiRoutes.when('/elasticsearch/ccr', {
         title: i18n.translate('xpack.monitoring.elasticsearch.ccr.routeTitle', {
           defaultMessage: 'Elasticsearch - Ccr',
         }),
+        pageTitle: i18n.translate('xpack.monitoring.elasticsearch.ccr.pageTitle', {
+          defaultMessage: 'Elasticsearch Ccr',
```

Comment:
I think we should use one/same title for `title` and `pageTitle`, and also `Ccr` should be `CCR`